x-pack/plugins/alerting_builtins/server/alert_types/index_threshold/alert_type.ts

```diff
@@ -67,6 +68,36 @@ export function getAlertType(service: Service): AlertType {
     }
   );
 
+  const actionVariableContextThresholdLabel = i18n.translate(
+    'xpack.alertingBuiltins.indexThreshold.actionVariableContextThresholdLabel',
+    {
+      defaultMessage:
+        "A values to use as the threshold; 'between' and 'notBetween' require two values, the others require one.",
+    }
+  );
+
+  const actionVariableContextThresholdComparatorLabel = i18n.translate(
+    'xpack.alertingBuiltins.indexThreshold.actionVariableContextThresholdComparatorLabel',
+    {
+      defaultMessage: 'A comparison function to use to determine if the threshold as been met.',
+    }
+  );
+
+  const alertParamsVariables = Object.keys(CoreQueryParamsSchemaProperties).map(
+    (propKey: string) => {
+      return {
+        name: propKey,
+        description: i18n.translate(
+          'xpack.alertingBuiltins.indexThreshold.actionVariableContextDateLabel',
+          {
+            defaultMessage: '{propKey}',
+            values: { propKey },
+          }
+        ),
```

Comment:
I'm trying to understand the difference between this and `description: propKey,`. From what I see, the `i18n.translate` function gets called for each `CoreQueryParamsSchemaProperties` and would return `propKey` value for each one.